protocol/development-workflow/README.md

```diff
@@ -56,43 +63,31 @@ Get the most recent state of the release branch.
     git checkout <release-name>
     git pull
 
-Create a QA branch off of `master`.
-
-    git checkout master
-    git pull
-    git checkout -b qa/<release-name>
-
-Merge the release branch into the QA branch.
-
-    git merge <release-name>
+Open a pull request to merge the release branch into `master`, triggering a Travis CI build to run our full test suite.
 
 Copy the QA steps from the Pivotal Tracker story into the PR description, and put the story ID in the PR title. Add the link to the PT story in the description of the PR.
 
-Determine which server is available on the [server spreadsheet](https://docs.google.com/spreadsheets/d/1qZ5x80cYXHxACJbZ20W5MqH6ETRFDmaLTnsjaqen6O0/edit), update the row of your choice, and push to that server.
+**Note**: After creating the PR, Github will let you know whether your release branch is up-to-date with master. If not, please use `git rebase` instead of the update feature on Github (which creates a merge commit).
 
-    git push <APP REMOTE> head:master --force
-
-Where <APP REMOTE> is the name of the remote you set up for the server of your choice. If you're not sure what's available, use `git remote -v` and refresh yourself by reading the [QA Setup](qa-setup.md) steps.
-
-Push the QA branch to GitHub.
+Wait for Travis CI to finish its build on the pull request.
 
-    git push -u origin qa/<release-name>
+Determine which server is available on the [server spreadsheet][server-spreadsheet]. Update the row of your choice, and push to that server.
 
-Open a pull request to merge the QA branch into `master`, triggering a Travis CI build to run our full test suite.
+    git push <APP REMOTE> head:master --force
 
-Wait for the staging delivery to finish.
+Where `<APP REMOTE>` is the name of the remote you set up for the server of your choice. If you're not sure what's available, use `git remote -v` and refresh yourself by reading the [QA Setup](qa-setup.md) steps.
 
-Conduct QA through your staging app's frontend. Reference the Server Spreadsheet to find the correct URL for your app.
+Follow the QA steps from Pivotal Tracker, conducting QA through your staging app's frontend, as needed. Reference the [Server Spreadsheet][server-spreadsheet] to find the correct URL for your app.
 
 If required by the specified QA steps, use the staging console.
 
     aptible ssh --app <APP NAME> bin/rails c
 
 ### Rejecting
 
-Close the QA branch's pull request on GitHub without merging.
+Close the QA pull request on GitHub without merging.
```

Comment:
Should we add that the developer should comment on the PT story?

"Close the QA pull request on GitHub without merging and notify the feature's engineer by posting a comment on the PT story, including reasons why the story did not pass QA."

Or something like that?

Reply:
Yes that should be in this guide. I'll add it today, and see if any other "comment on PT" stuff should be added too. Thank you!